.github/workflows/publish-technical-documentation-next.yml

```diff
@@ -0,0 +1,42 @@
+name: "publish-technical-documentation-next"
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "docs/sources/**"
+  workflow_dispatch:
```

Comment:
Is this property necessary? 👀

Reply:
Not strictly but I have found it is useful to be able to trigger the workflow in case of a spurious error without the need for a no-op docs change.